cli/asc.js

```diff
@@ -499,11 +499,14 @@ exports.main = function main(argv, options, callback) {
   // Initialize the program
   program = __pin(assemblyscript.newProgram(compilerOptions));
 
-  // Set up transforms
-  const transforms = [];
+  // Collect transforms *constructors* from the `--transform` CLI flag as well
+  // as the `transform` option into the `transforms` array.
+  let transforms = [];
+  // `transform` option from `main()`
   if (Array.isArray(options.transforms)) {
```

Comment:
I'm also wondering is normal that `options.transforms` and `opts.transform` [here](https://github.com/AssemblyScript/assemblyscript/pull/1814/files#diff-609c82113feed36cb75ab2bd3d387762bf2829fc5ae10477722fee8a97885489R510) have different key names? I'm not familiar with all this option keys

Reply:
Yeah, it is confusing (but as you can tell, that was already the case in the old code). I wanted to keep this PR minimal and not start renaming a lot of things.

`options` is the options object passed to `main()`, while `opts` is the return value of the command line flag parser. That’s why I added comments that one of the branches is about handling the transforms from the options object, while the other one handles transforms from the CLI flag.